css/style.css

```diff
@@ -1,6 +1,881 @@
+body {
+	margin: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)